test/unit/v1beta1/suggestion/test_chocolate_service.py

```diff
@@ -13,19 +13,20 @@
 # limitations under the License.
 
 import os
-
+import sys
 import grpc
 import grpc_testing
 import unittest
 import pytest
 
 from pkg.apis.manager.v1beta1.python import api_pb2
-
-from pkg.suggestion.v1beta1.chocolate.service import ChocolateService
+if sys.version_info < (3, 10):
+    from pkg.suggestion.v1beta1.chocolate.service import ChocolateService
 
 import utils
 
 
+@unittest.skipIf(sys.version_info >= (3, 10), reason="Chocolate supports only Python 3.9 or lower")
```

Comment:
We can not work chocolate on Python 3.10.

ref: https://github.com/kubeflow/katib/issues/2058

Reply:
Are we removing chocolate in this release itself? Else, removing tests will lead to untested releases

Reply:
@johnugeorge 
IIUC, we can run tests for chocolate in CI since I added Python 3.9 to CI so that tests for chocolate as shown below:

https://github.com/kubeflow/katib/blob/ff6f9139555b71d3894f652fa8204e4bc1f669db/.github/workflows/test-python.yaml#L29-L33

Although, I'm ok with removing chocolate in this release. 
wdyt?

Reply:
I agree, we should remove Chocolate first.
We want to reduce number of actions that we run since we have 20 concurrent GitHub Actions jobs limitation.

Reply:
Ok.
First, I will remove Chocolate Suggestion Service in another PR.